src/object/obj_tx.c

```diff
@@ -3273,6 +3273,10 @@ dc_tx_convert(struct dc_object *obj, enum obj_rpc_opc opc, tse_task_t *task)
 	int		 rc = 0;
 
 	D_ASSERT(obj != NULL);
+
+	if (task->dt_result == -DER_NEED_TX)
+		task->dt_result = 0;
```

Comment:
it looks not clear why can get "DER_NEED_TX" err to here, probably better to leave an ERROR log here because looks unreasonable to get such error code at this func. The bad possibility is there with some other bug related with task usage, directly reset to 0 may silently ignore it and cause other hard to find crash (not very sure how could get this err at here).